### PR TITLE
[EPO-3954] Allow pages to not require a hero

### PR DESCRIPTION
**Pull request by blnkt**

### CODE REVIEW

File: config/project.yaml [hunk 1]
```diff
@@ -1578,27 +1578,42 @@ fields:
     fieldGroup: d72dd4f7-1fcb-4812-bf47-c510d163bae8
     handle: defaultHeroImage
     instructions: ''
-    name: 'Hero Image'
+    name: 'Default Hero Image'
```
Comment (blnkt) [inline config/project.yaml:1581]: Ok. We're still having a misunderstand here I think.  What you've done is created a new set of fields called "Default Hero Image" and replaced the previous field "Hero Image" with your new set of fields.  That gives you some of the benefits in the templating that we're after, but it's not doing the job of consolidating our work.  

Please use the existing "Hero" set of fields (Type = matrix; Group = single pages).  Please also set the "Max Blocks" on the "Hero" to 1 (so a content editor cannot create multiple heroes).  Please delete the deprecated "Hero Image" field (I presuming it's never used elsewhere, do not delete it if it's used somewhere else) and your new "Default Hero Image" set of fields.  You will probably need to update some of the references to your global in the templates

**Review by blnkt — COMMENTED** [review]
very close, just looking for you to reuse the set of fields used throughout the site instead of creating a new set.

File: config/project.yaml (hunk 1 above)
Comment (georgegomez144) [reply config/project.yaml]: got it. it is done. Took me a bit to wrap my mind around your request, but I'm certain I understand you now.

Comment (blnkt) [reply config/project.yaml]: Great.  This is working as I would expect.  Thank you much.

**Pull request merged**